integration/experiment/gpu_frequency_sweep/gen_gpu_activity_constconfig_recommendation.py

```diff
@@ -89,5 +93,12 @@ def main(full_df):
         sys.exit(1)
 
     output = main(df)
-    sys.stdout.write("AGENT POLICY:\n")
-    sys.stdout.write(json.dumps(output) + "\n")
+
+    root_dir = os.getenv('GEOPM_SOURCE')
```

Comment:
What if this environment variable is not defined?

My understanding is that currently, the experiment infrastructure is meant to be used "in-tree". That being the case, if ``GEOPM_SOURCE`` is not provided, perhaps it's reasonable to craft the absolute path based on the script's current location (assuming where it sits relatively to the rest of the repo tree).

Reply:
GEOPM_SOURCE is provided as part of the .geopmrc file if a user is following the general setup.  In cases where it is not defined we error on a missing schema. 
```
FileNotFoundError: [Errno 2] No such file or directory: '/service/json_schemas/const_config_io.schema.json'
```

I'm open to moving to an absolute path however.